templates/docs/examples/patterns/forms/checkbox-validation.html

```diff
@@ -0,0 +1,18 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Forms / Checkbox Validation{% endblock %}
+
+{% block standalone_css %}patterns_form-tick-elements{% endblock %}
```

Comment:
This file `scss/standalone/patterns_form-tick-elements.scss` needs to be updated.

We require form validation now for example to work, so we need to add it. 
Probably `@include vf-p-form-validation;` will be enough.
You can check standalone versions to see if they look like standard examples:
https://vanilla-framework-5013.demos.haus/docs/examples/standalone/patterns/forms/radio-validation
https://vanilla-framework-5013.demos.haus/docs/examples/standalone/patterns/forms/checkbox-validation

All examples here: https://vanilla-framework-5013.demos.haus/docs/examples/standalone